styling-and-appearance/glyphs/common-styles-appearance-glyphs-reference-sheet.md

```diff
@@ -11,680 +11,690 @@ glyphs_reference_sheet: True
+|<i class=" t-i-border-radius-top-left telerik-fluent-icons-font-style"></i>|<i class=" t-i-border-radius-top-left"></i>|GlyphBorderRadiusTopLeft|e04e|&amp;#xe04e;|*|
+|<i class=" t-i-outline-width telerik-fluent-icons-font-style"></i>|<i class=" t-i-outline-width"></i>|GlyphOutlineWidth|e04f|&amp;#xe04f;|*|
+|<i class=" t-i-outline-offset telerik-fluent-icons-font-style"></i>|<i class=" t-i-outline-offset"></i>|GlyphOutlineOffset|e050|&amp;#xe050;|*|
+|<i class=" t-i-box-sizing telerik-fluent-icons-font-style"></i>|<i class=" t-i-box-sizing"></i>|GlyphBoxSizing|e051|&amp;#xe051;|*|
+|<i class=" t-i-position-top telerik-fluent-icons-font-style"></i>|<i class=" t-i-position-top"></i>|GlyphPositionTop|e052|&amp;#xe052;|*|
+|<i class=" t-i-position-right telerik-fluent-icons-font-style"></i>|<i class=" t-i-position-right"></i>|GlyphPositionRight|e053|&amp;#xe053;|*|
+|<i class=" t-i-position-bottom telerik-fluent-icons-font-style"></i>|<i class=" t-i-position-bottom"></i>|GlyphPositionBottom|e054|&amp;#xe054;|*|
+|<i class=" t-i-position-left telerik-fluent-icons-font-style"></i>|<i class=" t-i-position-left"></i>|GlyphPositionLeft|e055|&amp;#xe055;|*|
+|<i class=" t-i-display-block telerik-fluent-icons-font-style"></i>|<i class=" t-i-display-block"></i>|GlyphDisplayBlock|e056|&amp;#xe056;|*|
+|<i class=" t-i-display-inline-block telerik-fluent-icons-font-style"></i>|<i class=" t-i-display-inline-block"></i>|GlyphDisplayInlineBlock|e057|&amp;#xe057;|*|
+|<i class=" t-i-display-flex telerik-fluent-icons-font-style"></i>|<i class=" t-i-display-flex"></i>|GlyphDisplayFlex|e058|&amp;#xe058;|*|
+|<i class=" t-i-display-inline-flex telerik-fluent-icons-font-style"></i>|<i class=" t-i-display-inline-flex"></i>|GlyphDisplayInlineFlex|e059|&amp;#xe059;|*|
+|<i class=" t-i-gap-column telerik-fluent-icons-font-style"></i>|<i class=" t-i-gap-column"></i>|GlyphGapColumn|e05a|&amp;#xe05a;|*|
+|<i class=" t-i-gap-row telerik-fluent-icons-font-style"></i>|<i class=" t-i-gap-row"></i>|GlyphGapRow|e05b|&amp;#xe05b;|*|
+|<i class=" t-i-handle-resize-alt telerik-fluent-icons-font-style"></i>|<i class=" t-i-handle-resize-alt"></i>|GlyphHandleResizeAlt|e05c|&amp;#xe05c;|*|
+|<i class=" t-i-handle-resize telerik-fluent-icons-font-style"></i>|<i class=" t-i-handle-resize"></i>|GlyphHandleResize|e05d|&amp;#xe05d;|*|
+|<i class=" t-i-chevron-double-up telerik-fluent-icons-font-style"></i>|<i class=" t-i-chevron-double-up"></i>|GlyphChevronDoubleUp|e05е|&amp;#xe05е;|*|
+|<i class=" t-i-chevron-double-right telerik-fluent-icons-font-style"></i>|<i class=" t-i-chevron-double-right"></i>|GlyphChevronDoubleRight|e05f|&amp;#xe05f;|*|
+|<i class=" t-i-chevron-double-down telerik-fluent-icons-font-style"></i>|<i class=" t-i-chevron-double-down"></i>|GlyphChevronDoubleBottom|e060|&amp;#xe060;|*|
+|<i class=" t-i-chevron-double-left telerik-fluent-icons-font-style"></i>|<i class=" t-i-chevron-double-left"></i>|GlyphChevronDoubleLeft|e061|&amp;#xe061;|*|
 ||__Action__|__e100__|||
```

Comment:
The content in these "category" cells should be moved with one column to the right. Also, you should add an extra column here.

It should become:
`|||__Action__|__e100__|||`

Reply:
done